dae-alright/src/Redux/reducers/simpleReducer.js

```diff
@@ -0,0 +1,11 @@
+import SIMPLE_ACTION from "../constants/action-type"
```

Comment:
rename this file to `userReducer`, and create another `reducer` like this for `restaurant`

Reply:
Noted